cypress/integration/actions.js

```diff
@@ -173,10 +183,63 @@ describe('electronjs.org', () => {
   })
 
   describe('landing pages', () => {
-    // TODO(HashimotoYT): Add tests for Devtron and Spectron landing pages
     it('Fiddle landing page', () => {
       cy.visit(`${localhost}/fiddle`)
       cy.get('.jumbotron-lead').contains('The easiest way to get started with Electron')
     })
+
+    it('Devtron landing page', () => {
+      visit(`devtron`)
+      cy.get('.jumbotron-lead').contains('An Electron DevTools Extension')
+    })
+
+    it('Spectron landing page', () => {
+      visit(`spectron`)
+      cy.get('.jumbotron-lead').contains('An Electron Testing Framework')
+    })
+  })
+
+  describe('releases', () => {
+    it('/releases/stable', () => {
+      cy.visit(`${localhost}/releases/stable`)
+      cy.get('.r-resp-header-narrow').contains('Stable Releases')
+      cy.get('.release-entry').should('have.length', 5)
+      cy.get('a.releases-link-stable').should('have.class', 'active')
+    })
+
+    it('responsive navbar', () => {
+      cy.viewport('iphone-6', "portrait")
+      cy.visit(`${localhost}/releases/stable`)
+      cy.get('#release-navbar').should('have.class', 'd-none')
+      cy.get('.r-resp-header-toggle').should('be.visible').click()
+
+      // cy.get('#release-navbar').should('not.have.class', 'd-none')
+      // cy.get('#release-navbar > h3').should('be.visible').contains('Show Releases:')
```

Comment:
Should we do something with these?

Reply:
I'm trying to give chance for this today later if it's gone to be green all fine if red delete.